docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
 # Connect an æpp to a wallet
 
-## Introduction
+This guide describes the 4 steps that are necessary to connect your application to a wallet using the RPC API.
 
-This guide describes the 3 steps that are necessary to connect your application to a wallet using the RPC API.
+## Prerequisites
 
-Note:
-
-- The steps below are snippets taken from the full implementation of the [Simple æpp](https://github.com/aeternity/aepp-sdk-js/tree/master/examples/browser/aepp)
+- Install [Superhero Wallet extension](https://wallet.superhero.com/) for simplicity of example.
+You can build your own wallet in the next example
 
-## 1. Initialize the `RpcAepp` Stamp
+## 1. Specify imports and constants and state
 
 ```js
-import { RpcAepp, WalletDetector, BrowserWindowMessageConnection } from '@aeternity/aepp-sdk'
+import { RpcAepp, WalletDetector, BrowserWindowMessageConnection, Node } from '@aeternity/aepp-sdk'
+
+const TESTNET_NODE_URL = 'https://testnet.aeternity.io'
+const MAINNET_NODE_URL = 'https://mainnet.aeternity.io'
+const COMPILER_URL = 'https://compiler.aepps.com';
 
+export default {
+  data: () => ({
+    aeSdk: null,
+    publicAddress: '',
+    balance: 0,
+    node: null,
+    accounts: null
+  }),
+}
+```
+
+## 2. Initialize the `RpcAepp` Stamp
+
+```js
 async created () {
   this.aeSdk = await RpcAepp({
     name: 'Simple Aepp',
     nodes: [
-      { name: 'ae_uat', instance: await Node({ url: TEST_NET_NODE_URL }) },
-      { name: 'ae_mainnet', instance: await Node({ url: MAIN_NET_NODE_INTERNAL_URL }) }
+      { name: 'ae_uat', instance: await Node({ url: TESTNET_NODE_URL }) },
+      { name: 'ae_mainnet', instance: await Node({ url: MAINNET_NODE_URL }) }
     ],
     compilerUrl: COMPILER_URL,
-    onNetworkChange: async (params) => {
-      this.aeSdk.selectNode(params.networkId)
-      this.nodeInfoResponse = await this.aeSdk.getNodeInfo()
+    onNetworkChange: async ({ networkId }) => {
+      this.aeSdk.selectNode(networkId)
+      this.node = await this.aeSdk.getNodeInfo()
     },
-    onAddressChange:  async (addresses) => {
-      this.pub = await this.aeSdk.address()
-      this.balance = await this.aeSdk.balance(this.pub).catch(e => '0')
-      this.addressResponse = await this.aeSdk.address()
+    onAddressChange: async () => {
+      this.publicAddress = await this.aeSdk.address()
```

Comment:
`pub` -> `publicAddress` - I saw no good reason for shortening the name.

`this.addressResponse = await this.aeSdk.address()` was just duplicate

Reply:
I completely agree with no-shortening, but would name it just `address`, because we don't have any private addresses. Usually, I'm naming `ak_...` string an address, array of bytes a private/public key, but it is not aligned with naming in node.

Reply:
Good point, changed to `address`